terraform/projects/pangeo-hubs.tfvars

```diff
@@ -0,0 +1,30 @@
+prefix     = "pangeo-hubs"
+project_id = "pangeo-integration-te-3eea"
+
+core_node_machine_type = "n1-standard-1"
```

Comment:
@yuvipanda, IIRC, you used 3 g1-small in the meom-ige cluster... should we think about using those here as well?

Reply:
I've left some comments about that here: https://github.com/2i2c-org/pilot-hubs/blob/master/docs/topic/cluster-design.md#core-node-size. We can start with g1- and change if necessary?

Reply:
If I understand the docs correctly, we know we'll need a storage bucket and therefore will need to enable the config connector, which would lead us to an `n1-highmem-4` machine for the core pool?

Reply:
@sgibson91 Yeah, that makes sense! Let's do that!

Reply:
```suggestion
core_node_machine_type = "n1-highmem-4"
```

Reply:
I "think" you could have scratch buckets with metadata concealment... this is a single-tenant cluster, right?
So network policy and config connector should not be actually needed unless I am missing something (most likely!).
I was expecting the Pangeo cluster to follow meom one: https://github.com/2i2c-org/pilot-hubs/blob/master/terraform/projects/meom-ige.tfvars

Reply:
I think the current setup has staging and prod hub deployments on a single cluster, and then they'll eventually want their binder deployment pulling over too. So I think it's only temporarily a single tenant cluster 😄

Reply:
Just double checked: currently the hub(s) and binder are separate clusters, but the JupyterHub is running a staging and production deployment environment on a single cluster.

Reply:
> So I think it's only temporarily a single tenant cluster

OK, fair enough :wink: 
I can see JupyterHub and Binder working together in a multi-tenant cluster in the future.

Reply:
Yeah, and I think it makes sense for the Pangeo binder that is dask-enabled too. I think the work on closing the gap between the z2jh and binderhub charts will need to happen to make it feasible.

Reply:
Finally, IIRC, @yuvipanda mentioned two `n1-highcpu-2` nodes (in the meom cluster tfvars file) because you need at least two nodes when networkpolicy is enabled as it is enabled in this PR. I guess with 2 `n1-highmem-4` we are covered but wondering if we are not spending more resources than we need (sorry to be annoying with all of these, I guess it is only showcasing I do not have a clear picture yet about the real requirements for all the available options).

Reply:
Me either! That's why I opened this PR early! This is all really helpful :blush:

Reply:
I think the way to do this is to just build it up and see how it goes. We can check actual memory & CPU use, and adjust accordingly. Starting with `n1-highmem-4` is probably good :)

Reply:
Faur enough to me :wink: